training-material.md

```diff
@@ -13,7 +13,7 @@ The most recent training VM includes:
 
 ### TL;DR
 
-- Training VM system account : `thehive/thehive1234`
```

Comment:
Keep this line.

Reply:
this is why the point is named Training VM **system account**